### ISSUE TITLE
Cryptic error message when using an invalid API token

### ISSUE THREAD

**Issue by shibaobun** [reluce/szurubooru-toolkit#54]
Hello, I seem to be getting these error logs while trying to run the autotagger automatically in the docker container:

```
[INFO] Retrieving posts from https://szuru.bubbletea.dev with query " -deepbooru"...
[CRITICAL] [szurubooru.get_posts] Could not process your query: 'total'
```

Thank you!

**Comment by shibaobun** [comment]
Nvm this is my fault, I invalidated my token accidentally. It might be nice to add in a check for this though!

**Comment by reluce** [comment]
Thanks for the report, I pushed a quick fix to `main` which will be included in `1.0.1`. Cheers!
